.github/workflows/ci.yml

```diff
@@ -12,9 +12,15 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
+        java-version:
+          - "adopt@1.11"
+          - "adopt@1.8"
+          - "openjdk@1.11"
```

Comment:
What’s the motivation to have two JDK 11 entries?

Reply:
We should include the jdk name in the job name to make it easier to distinguish in the web ui.

Reply:
@olafurpg I'm fine with just one. I added two because I had noticed you were using AdoptOpenJDK for the 1.8 builds. I thought it may be wise to use OpenJDK's "official" implementation when available, which it is for 11, and I also wanted to keep parity with what you were already using, hence AdoptOpenJDK. 

That being said, using just one of the two is probably fine.